published/external/xdpapi.h

```diff
@@ -276,8 +339,8 @@ typedef struct _XDP_QEO_CONFIGURATION {
 typedef HRESULT
 XDP_QEO_SET_FN(
```

Comment:
Shouldn't we hide this behind some PRERELEASE C define?

Reply:
IDK...all of XDP is prerelease, right? We make breaking changes to our API at will, so I don't think we need to individually gate our APIs. If we did, we'd have a lot more APIs to gate.

Reply:
Yeah, I guess. Thanks.